forte/data/readers/sst2_reader.py

```diff
@@ -61,16 +61,18 @@ def _check_file_exist(self, filename: str):
             raise FileNotFoundError(
                 errno.ENOENT, os.strerror(errno.ENOENT), filename)
 
-    def _collect(self, *args, **kwargs) -> Iterator[DataPack]:
+    def _collect(self, *args, **kwargs) \
+        -> Iterator[List[Tuple[str, str, List[int]]]]:
         # pylint: disable = unused-argument
         r"""Iterator over sst files in the data_source.
         The directory should at least have the following files:
             "dictionary.txt": a mapping from phrase text to phrase id
             "sentiment_labels": a mapping from phrase id to sentiment score
             "datasetSentences.txt": original text for the sentence
-            "STree.txt": parent pointer list of constituency tree for each sentence
+            "STree.txt": parent pointer list of constituency tree
         Args:
             args: args[0] is the directory to the sst2 files.
+                args[1] is the number of sentences in a data pack
```

Comment:
Though this method is fine, Forte tend to use the configuration system, take a look at the example here:

https://github.com/asyml/forte/blob/master/forte/data/readers/openie_reader.py#L140